src/components/ChatLog.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = (props) => { 
+       
+    const chatDataMap = props.chats.map(item =>(
```

Comment:
using `item.id` as the key probably still gives you an error in Learn. You can get rid of this by using the index parameter for the map function like
```javascript 
const chatDataMap = props.chats.map((item, i) =>
<ChatEntry
       key={i}
```